doc/elements.md

```diff
@@ -1014,6 +1016,48 @@ Attribute | Type | Default | Description
 
 -----
 
+## `pl-drawing` element
+
+Creates a canvas (drawing space) that can display images from a collection of pre-defined drawing objects.
+Users can also add drawing objects to the canvas for grading.
```

Comment:
I suggest deleting all the text below this point and just having a sentence that explicitly says “See the pl-drawing documentation for details” (with a link).

Reply:
done.